framework_tvb/tvb/interfaces/web/controllers/flow_controller.py

```diff
@@ -522,13 +509,11 @@ def get_adapter_template(self, project_id, algorithm_id, is_upload=False, back_p
     @handle_error(redirect=True)
     @context_selected
     def reloadoperation(self, operation_id, **_):
-        """Redirect to Operation Input selection page,
-        with input data already selected."""
+        """Redirect to Operation Input selection page, with input data already selected."""
```

Comment:
This method was no longer working properly in case of Analyzers.
I think I just found the fix. Pls test

Reply:
tested and it seems to be working fine :)